### ISSUE TITLE
How to use facenet to recognize images of pets (cats and dogs)?

### ISSUE THREAD

**Issue by hi-karan-here** [davidsandberg/facenet#1037]
Hi David,

You work gives me hope that it is possible to play with facenet model using tensorflow. I am interested to know how can one recognize images of cats and dogs using facenet model.

I believe that pre-trained models won't be of any help here. What do you think?

Also, if I need to train something from scratch, how should I approach that?

Can you also throw some light for me on what is the training as a classifier and training using triplet loss function? I did find you mention that in the wiki section of this repo, but I am little confused.

Look forward to here from you.
